tools/cockpit.spec

```diff
@@ -65,6 +65,11 @@ ExcludeArch: %{ix86}
 %endif
 %endif
 
+%define enable_multihost 1
+%if 0%{?fedora} >= 41 || 0%{?rhel} >= 10
+%define enable_multihost 0
+%endif
```

Comment:
Feels a bit upside down now, but that's just bikeshedding.

Reply:
Yeah, `%if 0%{?fedora} <= 40 || 0%{?rhel} <= 9` doesn't work, unfortunately, since it is always true. One of `fedora` or `rhel` is not defined, which gives us one `0 <= N` condition, which is true. Unix...